package.json

```diff
@@ -4,7 +4,7 @@
   "description": "Utilities for runtime configuration validations",
   "main": "index.js",
   "scripts": {
-    "clean": "rimraf *.js",
+    "clean": "del *.js !*.config.js",
```

Comment:
why replace rimraf?

Reply:
Because it doesn't have an option to choose all the JS files except the config